diff --git a/dist/index.js b/dist/index.js
index 914285807221289f4a9b822fbcf18bec8e6da588..392c2d84cc95c26fa1adee2ca7c3b55b6b2dd9ba 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,5 @@
-import { getHighlighter } from 'shiki';
+// import { getHighlighter } from 'shiki'; // deprecated
+import { getSingletonHighlighter as getHighlighter } from 'shiki';
 import { visit } from 'unist-util-visit';
 import { toString } from 'hast-util-to-string';
 import rangeParser2 from 'parse-numeric-range';
